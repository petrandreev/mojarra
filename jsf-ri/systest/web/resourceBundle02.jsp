<%@ taglib uri="http://java.sun.com/jsf/html" prefix="h" %>
<%@ taglib uri="http://java.sun.com/jsf/core" prefix="f" %>

<f:view>Value from Faces component is <h:outputText
      value="#{resourceBundle01.value1}"/>.</f:view>
