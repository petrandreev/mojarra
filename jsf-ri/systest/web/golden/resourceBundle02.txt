


Value from Faces component is Value From ResourceBundle.
